src/style.css

```diff
@@ -1 +1,80 @@
-/* styles go here */
+/* stylelint-disable block-opening-brace-space-before */
```

Comment:
why you need all of this styles? please remove all and try to add only two classes for vertical distance
![image](https://user-images.githubusercontent.com/54936044/228381761-a3d8dd97-6dbc-4ed5-b0da-c9ed0657dfa8.png)